web-client/src/presenter/computeds/selectDocumentTypeHelper.ts

```diff
@@ -149,11 +149,23 @@ export const getPreviouslyFiledDocuments = (
     return { ...doc, documentTitle };
   };
 
-  return caseDetail.docketEntries
+  const formattedCaseDetail = applicationContext
+    .getUtilities()
+    .getFormattedCaseDetail({
+      applicationContext,
+      caseDetail,
+    });
```

Comment:
We already have logic on this function to determine is a document has been served or not.